shared/src/main/i18n/com/kuvaszuptime/kuvasz/i18n/Messages_fr.properties

```
@@ -0,0 +1,216 @@
+uptimeBlockTitle=Uptime
+recentEventsBlockTitle=Evènements récents
+sslBlockTitle=SSL
+latencyBlockTitle=Latence
+latencyBlockSubtitle=dans les dernières 24h
+latencyChartNoData=Aucune donnée pour l''instant
+latencyAverage=Moyenne
+recentMeasurements=Mesures récentes
+status=Status
+startedAt=Démarré le
+duration=Durée
+details=Details
+reasonExplanation=Raison: {0}
+untilDatetime=Depuis: {0}
+atDatetime=A: {0}
+sslStatusNotAvailable=Le status SSL n''est pas disponible
+lastCheck=Dernière vérification
+nextCheck=Prochaine vérification
+nextUptimeCheck=Prochaine vérification uptime
+currentlyFor=En cours pour {0}
+monitorWasJustCreated=Moniteur créé
+currentUptimeStatus=Status actuel de l''uptime
+configure=Configurer
+configuration=Configuration
+# Badges
+validUntil=Valide depuis: {0}
+valid=Valide
+invalid=Invalide
+expiresSoon=Expire bientôt
+expiresSoonValidUntil=Expire bientôt, valide jsqu''à: {0}
+waitingForCheck=En attente de la première vérification
+disabled=Désactivé
+unknownError=Erreur inconnue
+inProgress=En cours
+paused=Pause
+monitorIsPaused=Moniteur en pause
+# Intervals & Time
+inSeconds=dans {0} secondes
+secondsAgo=il y a {0} secondes
+justNow=juste maintenant
+inAMinute=dans une minute
+aMinuteAgo=il y a une minute
+inMinutes=dans {0} minutes
+minutesAgo=il y a {0} minutes
+inAnHour=dans une heure
+anHourAgo=il y a une heure
+inHours=dans {0} heures
+hoursAgo=il y a {0} heures
+inADay=dans un jour
+aDayAgo=il y a un jour
+inDays=dans {0} jours
+daysAgo=il y a {0} jours
+inAWeek=dans une semaine
+aWeekAgo=il y a une semaine
+inWeeks=dans {0} semaines
+weeksAgo=il y a {0} semaines
+inAMonth=dans un mois
+aMonthAgo=il y a un mois
+inMonths=dans {0} mois
+monthsAgo=il y a {0} mois
+inAYear=dans un an
+aYearAgo=il y a un an
+inYears=dans {0} ans
+yearsAgo=il y a {0} ans
+dayInterval={0} jour
+daysInterval={0} jours
+hourInterval={0} heure
+hoursInterval={0} heures
+minuteInterval={0} minute
+minutesInterval={0} minutes
+secondInterval={0} seconde
+secondsInterval={0} secondes
+xDay={0} jour
+xDays={0} jours
+xSecond={0} seconde
+xSeconds={0} secondes
+# Event formatters
+validSSLMessage=Votre site "{0}" ({1}) a un certificat valide
+# Generic stuff
+enabled=Activé
+# Email notifications
+hasAValidCertificate=a un certificat valide
+hasAnInvalidCertificate=a un certificat invalide
+hasAnExpiringCertificate=a un certificat qui expire bientôt
+# Formatters
+yourMonitorIsDown=Votre moniteur "{0}" ({1}) est DOWN{2}
+yourMonitorIsUp=Votre moniteur "{0}" ({1}) est UP ({2})
+latencyIs=Latence: {0}ms
+wasUpFor=UP depuis {0}
+wasDownFor=DOWN depuis {0}
+requestHasBeenRedirected=La requête vers "{0}" ({1}) a été redirigée vers {2}
+yourSiteHasAValidCert=Votre site "{0}" ({1}) a un certificat valide
+yourSiteHasAnInvalidCert=Votre site "{0}" ({1}) a un certificat invalide
+yourCertWillExpireSoon=Le certificat SSL pour {0} expire bientôt
+expiryDate=Date d''expiration: {0}
+wasXForY=C''était {0} pour {1}
+redacted=... [MASQUE]
```

```suggestion
redacted={0}... [MASQUE]
```

just changed this yesterday, because the translation itself wasn't used before, so I didn't realise that the parameter was actually missing from it